components/mixins/studentStatus.js

```diff
@@ -0,0 +1,10 @@
+export const stuStatus = {
+  methods: {
+    getStatus(isActive, isConcluding, isGraduating, isForming, isFit) {
```

Comment:
Muitos parâmetro. É interessante passar como parâmetros nomeados.

https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6

Reply:
Mudei já. Ficou elegante, mas, acho que isso pode gerar uma insegurança. Talvez não na nossa aplicação hoje, mas tipo, estou enviando mais informações a essa função do que o necessário...